man/systemd.socket.xml

```diff
@@ -429,6 +429,11 @@
         is the same as the format used by CGI. For <constant>SOCK_RAW</constant>, the port is the IP
         protocol.</para>
 
+        <para>For <constant>AF_UNIX</constant> socket connections, the <varname>REMOTE_ADDR</varname>
```

Comment:
```suggestion
        <para>For <constant>AF_UNIX</constant> socket connections, the <varname>$REMOTE_ADDR</varname>
```

Not your fault, but please apply this to the paragraph above as well.

Reply:
Done, also explained that $REMOTE_ADDR won't be set if socket is unnamed.